plugin/manager.go

```diff
@@ -35,7 +35,7 @@ func NewCLIManager(pluginFS dir.SysFS) *CLIManager {
 // Get returns a plugin on the system by its name.
 //
 // If the plugin is not found, the error is of type os.ErrNotExist.
-func (m *CLIManager) Get(ctx context.Context, name string) (Plugin, error) {
+func (m *CLIManager) Get(ctx context.Context, name string, pluginConfig map[string]string) (Plugin, error) {
 	// validate file existence
 	pluginPath := path.Join(name, binName(name))
 	fi, err := fs.Stat(m.pluginFS, pluginPath)
```

Comment:
Should the operation of validating file existence in `NewCLIPlugin()` instead of `CLIManager.Get()`?

Reply:
Updated. Moved to NewCLIPlugin()